tekton/release-cheat-sheet.md

```diff
@@ -0,0 +1,154 @@
+
+This doc is a condensed version of our [tekton/README.md](./README.md) as
+well as instructions from
+[our plumbing repo](https://github.com/tektoncd/plumbing/tree/master/tekton/resources/release/README.md#create-draft-release).
+
+These steps provide a no-frills guide to performing an official release
+of Tekton Pipelines. To follow these steps you'll need a checkout of
+the pipelines repo, a terminal window and a text editor.
+
+1. `cd` to root of Pipelines git checkout.
+
+2. Point `kubectl` at dogfooding cluster.
+
+    ```bash
+    gcloud container clusters get-credentials dogfooding --zone us-central1-a --project tekton-releases
+    ```
+
+3. Edit `tekton/resources.yaml`. Add pipeline resource for new version.
+
+    ```yaml
+    apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: # UPDATE THIS. Example: tekton-pipelines-v0-11-2
+    spec:
+      type: git
+      params:
+      - name: url
+        value: https://github.com/tektoncd/pipeline
+      - name: revision
+        value: # UPDATE THIS. Example: 33e0847e67fc9804689e50371746c3cdad4b0a9d
+    ```
+
+4. `kubectl apply -f tekton/resources.yaml`
+
+5. Create environment variables for bash scripts in later steps.
+
+    ```bash
+    VERSION_TAG=# UPDATE THIS. Example: v0.11.2
+    GIT_RESOURCE_NAME=# UPDATE THIS. Example: tekton-pipelines-v0-11-2
+    IMAGE_REGISTRY=gcr.io/tekton-releases
+    ```
+
+6. Confirm commit SHA matches what you want to release.
+
+    ```bash
+    kubectl get pipelineresource "$GIT_RESOURCE_NAME" -o=jsonpath="{'Target Revision: '}{.spec.params[?(@.name == 'revision')].value}{'\n'}"
+    ```
+
+7. Execute the release pipeline.
+
+    **If you are backporting include this flag: `--param=releaseAsLatest="false"`**
+
+    ```bash
+    tkn pipeline start \
+      --param=versionTag="${VERSION_TAG}" \
+      --param=imageRegistry="${IMAGE_REGISTRY}" \
+      --serviceaccount=release-right-meow \
+      --resource=source-repo="${GIT_RESOURCE_NAME}" \
+      --resource=bucket=pipeline-tekton-bucket \
+      --resource=builtBaseImage=base-image \
+      --resource=builtEntrypointImage=entrypoint-image \
+      --resource=builtKubeconfigWriterImage=kubeconfigwriter-image \
+      --resource=builtCredsInitImage=creds-init-image \
+      --resource=builtGitInitImage=git-init-image \
+      --resource=builtControllerImage=controller-image \
+      --resource=builtWebhookImage=webhook-image \
+      --resource=builtDigestExporterImage=digest-exporter-image \
+      --resource=builtPullRequestInitImage=pull-request-init-image \
+      --resource=builtGcsFetcherImage=gcs-fetcher-image \
+      --resource=notification=post-release-trigger \
+    pipeline-release
+    ```
+
+8. Watch logs of pipeline-release.
+
+9. The YAMLs are now released! Anyone installing Tekton Pipelines will now get the new version. Time to create a new GitHub release announcement:
+
+    1. Create TaskRun YAML file to execute create-draft-release Task.
```

Comment:
Note: this process differs from [our plumbing repo](https://github.com/tektoncd/plumbing/tree/master/tekton/resources/release/README.md#create-draft-release) because right now the official version of `tkn` (0.8.0) will error out while trying to create the create-draft-release TaskRun.  I think this is happening because the create-draft-release Task is v1beta1 and `tkn` doesn't yet read the params at the top level of the Task spec in v1beta1.

Reply:
I was wondering what the tkn command was not working while trying to release Triggers!